docs/deployment.md

```
@@ -0,0 +1,31 @@
+
+At the end of a sprint a developer builds a release in the source code project. It needs to be done for each folder frontend/backend/eai separately. Then the devops engineer needs to update the chart-version and app-version of the helm chart in the Chart.yaml.
+
+Internal the quay syncs the docker images with the external github image registry. The github api has a pull limit so we need to cache the images internally.
+
+![image](images/internal-chart.drawio.png)
+
+The internal gitlab repo you can config your application, for example the trust store, ingress (openshift route), single sign on, database config, environment configs. The most config is comment out, so you need to comment it in. Here you see an example https://git.muenchen.de/ccse/cicd/sps-github
+
+
+You have two possibilities for the image rollout. 
+-	For the dev environment, you want to have an auto rollout. We create a image stream. The image stream has a link to the quay. In the deployment there is a special annotation set. Openshift then triggers automatically trigger a new rollout if a new image is available.  https://docs.redhat.com/en/documentation/openshift_container_platform/4.17/html/images/triggering-updates-on-imagestream-changes#triggering-updates-on-imagestream-changes
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 2

(MD010, no-hard-tabs)